tests/functional/s3curl/tests.js

```diff
@@ -865,55 +865,133 @@ describe('s3curl object ACLs', () => {
+        ], () => deleteFile(upload, done));
     });
 
     it('should list parts of multipart upload with no parts', done => {
         provideRawOutput([
             '--',
-            '-X',
-            'POST',
-            `${bucketPath}/${key}?uploads`,
+            `${bucketPath}/${key}?uploadId=${uploadId}`,
             '-v',
         ], (httpCode, rawOutput) => {
+            assert.strictEqual(httpCode, '200 OK');
             parseString(rawOutput.stdout, (err, result) => {
-                if (err) {
-                    assert.ifError(err);
-                }
-                uploadId = result.InitiateMultipartUploadResult.UploadId[0];
-                provideRawOutput([
-                    '--',
-                    `${bucketPath}/${key}?uploadId=${uploadId}`,
-                    '-v',
-                ], (httpCode, rawOutput) => {
+                assert.strictEqual(result.ListPartsResult.UploadId[0],
+                                   uploadId);
+                assert.strictEqual(result.ListPartsResult.Bucket[0],
+                                   bucket);
+                assert.strictEqual(result.ListPartsResult.Key[0], key);
+                assert.strictEqual(result.ListPartsResult.Part,
+                                   undefined);
+                done();
+            });
+        });
+    });
+
+    it('should copy a part and return lastModified as ISO', done => {
+        provideRawOutput(
+            ['--', `${bucketPath}/${key}?uploadId=${uploadId}&partNumber=1`,
+            '-X', 'PUT', '-H',
+            `x-amz-copy-source:${bucket}/copyme`, '-v'],
+            (httpCode, rawOutput) => {
+                assert.strictEqual(httpCode, '200 OK');
+                parseString(rawOutput.stdout, (err, result) => {
+                    const lastModified = result.CopyPartResult
+                        .LastModified[0];
+                    const isoDateString = new Date(lastModified).toISOString();
+                    assert.strictEqual(lastModified, isoDateString);
+                    done();
+                });
+            });
+    });
+});
+
+describe('s3curl copy object', () => {
+    before(done => {
+        createFile(upload, 1048576, () => {
+            provideRawOutput(
+                ['--createBucket', '--', bucketPath, '-v'],
+                httpCode => {
                     assert.strictEqual(httpCode, '200 OK');
-                    parseString(rawOutput.stdout, (err, result) => {
-                        assert.strictEqual(result.ListPartsResult.UploadId[0],
-                                           uploadId);
-                        assert.strictEqual(result.ListPartsResult.Bucket[0],
-                                           bucket);
-                        assert.strictEqual(result.ListPartsResult.Key[0], key);
-                        assert.strictEqual(result.ListPartsResult.Part,
-                                           undefined);
+                    provideRawOutput([
```

Comment:
I thought you were referring to the copy one below.  This one can be switched out.